lib/Modules/OVSCharmUtils/OVSCharmUtils.psm1

```diff
@@ -281,47 +293,66 @@ function Disable-OVS {
             $ovsRunning = $false
         }
     }
-
     if($ovsRunning) {
         $bridges = Start-ExternalCommand { & $OVS_VSCTL list-br }
         foreach($bridge in $bridges) {
             Start-ExternalCommand { & $OVS_VSCTL del-br $bridge }
         }
     }
-
     foreach($svcName in $ovsServices) {
         $svc = Get-Service -Name $svcName -ErrorAction SilentlyContinue
         if($svc) {
             Stop-Service $svcName -Force
             Disable-Service $svcName
         }
     }
-
     Disable-OVSExtension
 }
 
 function Enable-OVS {
     Enable-OVSExtension
-
     $ovsServices = @($OVS_OVSDB_SERVICE_NAME, $OVS_VSWITCHD_SERVICE_NAME)
     foreach($svcName in $ovsServices) {
         Enable-Service $svcName
         Start-Service $svcName
     }
+    Invoke-JujuCommand -Command @($OVS_VSCTL, 'set-manager', 'ptcp:6640:127.0.0.1')
```

Comment:
We are able now to use the `native` OVS interface for `neutron-ovs-agent`. See: https://github.com/cloudbase/neutron/commit/db129566aca26bf9d414066574e3c4a9b2479097

But this need to bet enabled as a prerequisite.